### ISSUE TITLE
Cmus widget doesn't update when song title has an ampersand ("&")

### ISSUE THREAD

**Issue by botanicEventuality** [qtile/qtile#2649]
# Issue description
Whenever a song comes up that has an ampersand in the title, the cmus widget stops updating, even after the song in particular ends. I don't know if it happens with any other symbols, or if it happens when the ampersand is in the artist name. Restarting Qtile fixes the issue as long as the current song does not have an ampersand in its title.

# Qtile version
0.18.1.dev0+g8e7ecc0a.d20210719

# Stack traces
Looking into `~/.local/share/qtile/qtile.log` shows the following

```
2021-07-23 22:20:47,453 ERROR libqtile base.py:on_done():L563 Failed to reschedule.
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 555, in on_done
    self.update(result)
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 467, in update
    self.text = text
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 359, in text
    self.layout.text = self.formatted_text
  File "/usr/lib/python3.9/site-packages/libqtile/drawer.py", line 71, in text
    attrlist, value, accel_char = pangocffi.parse_markup(value)
  File "/usr/lib/python3.9/site-packages/libqtile/pangocffi.py", line 183, in parse_markup
    raise Exception("parse_markup() failed for %s" % value)
Exception: parse_markup() failed for b'\xe2\x99\xab Daisuke Ishiwatari - SHOTGUN&HEAD'
2021-07-23 22:22:22,656 WARNING libqtile lifecycle.py:_atexit():L34 Restarting Qtile with os.execv(...)
2021-07-23 22:23:32,624 ERROR libqtile base.py:on_done():L563 Failed to reschedule.
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 555, in on_done
    self.update(result)
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 467, in update
    self.text = text
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 359, in text
    self.layout.text = self.formatted_text
  File "/usr/lib/python3.9/site-packages/libqtile/drawer.py", line 71, in text
    attrlist, value, accel_char = pangocffi.parse_markup(value)
  File "/usr/lib/python3.9/site-packages/libqtile/pangocffi.py", line 183, in parse_markup
    raise Exception("parse_markup() failed for %s" % value)
Exception: parse_markup() failed for b'\xe2\x99\xab Homestuck - Walk-Stab-Walk (R&E)'
2021-07-23 22:23:32,628 ERROR libqtile base.py:on_done():L563 Failed to reschedule.
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 555, in on_done
    self.update(result)
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 467, in update
    self.text = text
  File "/usr/lib/python3.9/site-packages/libqtile/widget/base.py", line 359, in text
    self.layout.text = self.formatted_text
  File "/usr/lib/python3.9/site-packages/libqtile/drawer.py", line 71, in text
    attrlist, value, accel_char = pangocffi.parse_markup(value)
  File "/usr/lib/python3.9/site-packages/libqtile/pangocffi.py", line 183, in parse_markup
    raise Exception("parse_markup() failed for %s" % value)
Exception: parse_markup() failed for b'\xe2\x99\xab Homestuck - Walk-Stab-Walk (R&E)'
```

**Comment by elParaguayo** [comment]
Fixed the formatting for you.

Thanks for the bug report. I'll take a look at this and see if I can fix it.

**Comment by elParaguayo** [comment]
Can you try the fix in #2650 and let me know if it works.

**Comment by botanicEventuality** [comment]
> Can you try the fix in #2650 and let me know if it works.

It works! Thanks.
I don't know if it's because I was running it on a second X Session alongside the version of Qtile I had installed already, but I had to restart it for the Cmus widget to show up.
